tsdb/wlog/watcher_test.go

```diff
@@ -550,7 +550,8 @@ func TestCheckpointSeriesReset(t *testing.T) {
 		segments int
 	}{
 		{compress: CompressionNone, segments: 14},
-		{compress: CompressionSnappy, segments: 13},
+		{compress: CompressionSnappy, segments: 14},
```

Comment:
Can you comment on why this changed?

Reply:
After changing from golang/snappy to klauspost/compress/snappy, this test started failing, so to fix that i had to change it to 14. Should snappy compression have lesser number of segments than one without compression.

Reply:
It does seem concerning if the data size goes up.
Can you investigate whether this is a general pattern?  Maybe the library author has advice?

Reply:
Sure i will do some investigation about it.